src/Plugin/EdgeKeyTypeBase.php

```diff
@@ -166,4 +170,11 @@ public function getAccountKey(KeyInterface $key): array {
     return $json;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function useGcpDefaultServiceAccount(KeyInterface $key): bool {
+    return $key->getKeyValues()['gcp_hosted'] ? TRUE : FALSE;
```

Comment:
Using `empty` prevents throwing a notice if the value is not set on the key.
```suggestion
    return $key->getKeyValues()['gcp_hosted'] ? TRUE : FALSE;
    return !empty($key->getKeyValues()['gcp_hosted']);
```

Reply:
Thank you I fixed this.